composer.json

```diff
@@ -12,7 +12,7 @@
         "psr-4": { "DeepCopyTest\\": "tests/DeepCopyTest/" }
     },
     "require": {
-        "php": ">=5.4.0"
```

Comment:
This is not related to the Travis build, and I'm not sure about following this policy. I'm aware of BC breaks in PHP 7.1 and that it doesn't follow SemVer, but it's a global question: packages doing this that are not updated when new versions of PHP are released will not be usable and will probably block many users. This may make a lot of people unhappy in the future and I'm currently on the fence on applying it everywhere or nowhere.

I think a first step would be to remove it so that the PR is only about improving CI, and maybe open a separate PR/issue about this to discuss it. I'd be interested to know how other projects are approaching this?